repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/git/GitHelper.java

```diff
@@ -337,19 +337,39 @@ public void gitAdd(List<String> files, Git git) {
         }
     }
 
-    public void copyDirectory(File sourceDir, File targetDir, String[] excludedFileNames, AbstractStep step) {
+    /**
+     * Copy the files from a directory to another.
+     *
+     * @param sourceDir is the directory containing the files to be copied from.
+     * @param targetDir is the directory where the files from sourceDir are going to be copied to.
+     * @param excludedFileNames is a list of file names not to be included in targetDir.
+     * @param isToRemove when set as "true", all files will be copied from sourceDir to targetDir, and after that files with the names in excludedFileNames will be removed from targetDir.
```

Comment:
Reading the beginning of the doc it looks like it's about the process of copying/deleting file, but at the end it looks more about the way the previous argument is used.

If I follow you, `isToRemove` is only used to know if the `excludedFileName` should correspond exactly to a fileName or if it should be used as a pattern / a substring of a filename. Right?

Reply:
I understand your point, and it means to me I should change stuff. I will do changes in both code and doc and push it again.